src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -26,25 +26,38 @@
     /** Should show consumption list. */
     private final boolean isShowConsumption;
 
+    /** Should show edit ingredient command. */
+    private final boolean isEditIngredient;
+
+    /** Should show edit recipe command. */
+    private final boolean isEditRecipe;
+
+    private String commandBoxText;
+
+
     /**
      * Constructs a {@code CommandResult} with the specified fields.
      */
     public CommandResult(String feedbackToUser, boolean showHelp, boolean exit,
-                         boolean isShowRecipe, boolean isShowIngredient, boolean isShowConsumption) {
+                         boolean isShowRecipe, boolean isShowIngredient, boolean isShowConsumption,
+                         boolean isEditRecipe, boolean isEditIngredient) {
         this.feedbackToUser = requireNonNull(feedbackToUser);
         this.showHelp = showHelp;
         this.exit = exit;
         this.isShowRecipe = isShowRecipe;
         this.isShowIngredient = isShowIngredient;
         this.isShowConsumption = isShowConsumption;
+        this.isEditRecipe = isEditRecipe;
+        this.isEditIngredient = isEditIngredient;
     }
 
     /**
      * Constructs a {@code CommandResult} with the specified {@code feedbackToUser},
      * and other fields set to their default value.
      */
     public CommandResult(String feedbackToUser) {
-        this(feedbackToUser, false, false, false, false, false);
```

Comment:
Instead of passing many booleans, you can just pass in a string e.g if its to show recipes "show recipes" then do a switch statement to set the attribute to true, every other boolean will be default false